spec/models/post_spec.rb

```diff
@@ -1,5 +1,60 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
There are still some tests failing. It is because there are some syntax errors in test definitions. Kindly see the screenshot below.
![test_errors](https://user-images.githubusercontent.com/35504149/167270323-131b50bd-6e9b-4161-ab2b-1c67ed27edc4.png)

Reply:
What could be the issue please, because on my end all the specs are passing :heavy_check_mark: ?

Reply:
@SajjadAhmad14  could you please participate in this review ? There is no way I can make progress if I don't get consistent feedback from you.